Formula/visp.rb

```diff
@@ -46,7 +40,7 @@ def install
                          "-DUSE_JPEG=ON",
                          "-DJPEG_INCLUDE_DIR=#{Formula["jpeg"].opt_include}",
                          "-DJPEG_LIBRARY=#{Formula["jpeg"].opt_lib}/libjpeg.dylib",
-                         "-DUSE_LAPACK=OFF",
+                         "-DUSE_LAPACK=ON",
```

Comment:
Looks like that was a link to an older debian library and it should be https://github.com/lagadic/visp/blob/50d1477c44699421dc9b80d4305d579f059e0201/CMakeLists.txt#L522-L523

Reply:
ViSP has dedicated functionalities based on Lapack, Eigen3 and GSL. All these 3rd parties could be used simultaneously, not only one of them. Depending on the matrice/vector size, an operation done with lapack could be faster than with GSL or EIgen3, or vice versa.
That's why in `vpMatrix` [class](https://visp-doc.inria.fr/doxygen/visp-daily/classvpMatrix.html) we have generic functions like [pseudoInverse](https://visp-doc.inria.fr/doxygen/visp-daily/classvpMatrix.html#ac324e74b23cc9fa9b132bb7085978714)() or functions that are using a specific 3rd party like [pseudoInverseLapack](https://visp-doc.inria.fr/doxygen/visp-daily/classvpMatrix.html#ac324e74b23cc9fa9b132bb7085978714)(), [pseudoInverseEigen3](https://visp-doc.inria.fr/doxygen/visp-daily/classvpMatrix.html#ad90e70d9cc8f9606944f25e88b345cc5)(), [pseudoInverseGsl](https://visp-doc.inria.fr/doxygen/visp-daily/classvpMatrix.html#a4a406e81301ebfc567056fb23bf074d4)(). The goal is that the user can compare the results and use the one that has the best performances.

Concerning lapack, I understand that since lapack is a system library, there is no need to add an explicit dependency:
```
  depends_on "lapack"
```
and specific cmake options
```
                         "-DUSE_LAPACK=ON",
                         "-DLAPACK_INCLUDE_DIR=#{Formula["lapack"].opt_include}",
                         "-DLAPACK_LIBRARY=#{Formula["lapack"].opt_lib}/liblapack.dylib",
```
Let me know if I'm wrong...